docs/pages/enroll-resources/database-access/enroll-google-cloud-databases/alloydb.mdx

```diff
@@ -0,0 +1,307 @@
+account. In the sidebar, choose "Cloud IAM" authentication type and add the
+"alloydb-user" service account that you created in
+[the second step](#step-29-create-a-service-account-for-a-database-user):
+
+![Add AlloyDB User Account](../../../../img/database-access/guides/alloydb/add-user-account.png)
+
+Press "Add" and your Users table should look similar to this:
+
+![AlloyDB User Accounts Table](../../../../img/database-access/guides/alloydb/user-account-added.png)
+
+See [Manage IAM authentication](https://cloud.google.com/alloydb/docs/database-users/manage-iam-auth)
+in Google Cloud documentation for more info.
+
+## Step 4/9. Install Teleport
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 5/9. Configure the Teleport Database Service
+
+### Create a join token
+
+(!docs/pages/includes/tctl-token.mdx serviceName="Database" tokenType="db" tokenFile="/tmp/token"!)
+
+### Generate Teleport config
+
+Provide the following information and then generate a configuration file for the
+Teleport Database Service:
+
+- <Var name="example.teleport.sh:443" /> The host and port of your Teleport
+  Proxy Service or Enterprise Cloud site.
+- <Var name="connection-uri" /> The AlloyDB connection URI for your instance in the form:
+  `projects/PROJECT/locations/REGION/clusters/CLUSTER/instances/INSTANCE`. 
+  
+  
+You can copy the connection URI from the AlloyDB instance details page.
+
+![AlloyDB Connection URI](../../../../img/database-access/guides/alloydb/connection-uri.png)
+
+Run the command as follows. Make sure to include the mandatory `alloydb://` prefix in the specified URI.
+
+```code
+$ sudo teleport db configure create \
+   -o file \
+   --name=alloydb \
+   --protocol=postgres \
+   --labels=env=dev \
+   --token=/tmp/token \
+   --proxy=<Var name="example.teleport.sh:443" />  \
+   --uri=alloydb://<Var name="full-instance-name" />
+```
+
+This command will generate a Teleport Database Service configuration file and
+save it to `/etc/teleport.yaml`.
+
+<Admonition title="Choose how Teleport connects to AlloyDB">
+By default, Teleport uses [_private_](https://cloud.google.com/alloydb/docs/about-private-services-access) AlloyDB endpoint. To change this to either [public](https://cloud.google.com/alloydb/docs/connect-public-ip) or [PSC](https://cloud.google.com/alloydb/docs/about-private-service-connect) endpoints, update the `endpoint_type` field:
+
+```yaml
+db_service:
+  resources:
+    - name: alloydb
+      protocol: postgres
+      uri: alloydb://projects/PROJECT/locations/REGION/clusters/CLUSTER/instances/INSTANCE
+      gcp:
+        alloydb:
+          # one of: private | public | psc (default: private)
+          endpoint_type: private
+```
+</Admonition>
+
+## Step 6/9. Configure GCP credentials
+
+The Teleport Database Service must have credentials for the
+`teleport-db-service` GCP service account.
+
+If the Teleport Database Service is hosted on a GCE instance, you can
```

Comment:
For convenience, is there a quick `gcloud` command for changing the instance's service account that we could include here?

Reply:
We don't really use `gcloud` for anything else on this page, so it is a somewhat weird outlier. Still, I've added details on how to do this through web UI and `gcloud` both.